vdr/vdr.go

```diff
@@ -150,7 +152,7 @@ func (r VDR) Update(id did.DID, current hash.SHA256Hash, next did.Document, _ *t
 		return err
 	}
 
-	keyID := controllers[0].Authentication[0].ID.String()
+	keyID := controllers[0].CapabilityInvocation[0].ID.String()
```

Comment:
should the alterating controller be specified in the future? Maybe we're missing a TODO/issue here.

The controller might not be controlled by the local Nuts node (edge case though).

Reply:
the `OwnedByThisNode` helper should fix that